husky_components/datagrid/decorators/table-view.js

```diff
@@ -286,10 +292,29 @@ define(function() {
         },
 
         /**
+         * Indents the datagrid's selected-counter
+         */
+        indentSelectedCounter: function() {
+            if (this.datagrid.options.selectedCounter === true) {
+                this.sandbox.dom.addClass(this.datagrid.$find('.selected-elements'), constants.counterIndentClass);
+            }
+        },
+
+        /**
+         * Removes the indent of the selected-counter
+         */
+        removeIndentSelectedCounter: function() {
+            if (this.datagrid.options.selectedCounter === true) {
+                this.sandbox.dom.removeClass(this.datagrid.$find('.selected-elements'), constants.counterIndentClass);
+            }
+        },
+
+        /**
          * Destroys the view
          */
         destroy: function() {
             this.sandbox.stop(this.sandbox.dom.find('*', this.$el));
+            this.removeIndentSelectedCounter();
```

Comment:
Is this necessary? The entire element gets destroyed afterwards anyway, doesn't it?